Mathlib/RingTheory/Valuation/Integers.lean

```diff
@@ -133,8 +135,68 @@ theorem le_iff_dvd {x y : O} : v (algebraMap O F x) ≤ v (algebraMap O F y) ↔
   ⟨hv.dvd_of_le, hv.le_of_dvd⟩
 #align valuation.integers.le_iff_dvd Valuation.Integers.le_iff_dvd
 
+theorem isUnit_of_one' {x : O} (hvx : v (algebraMap O F x) = 1) :
```

Comment:
Can you add a brief docstring describing the difference between this and the unprimed version?

Reply:
OK, I'll add some docstring for both this theorem and the unprimed version, explaining their relation and the theorem itself.